src/kw_remote.sh

```diff
@@ -210,6 +215,18 @@ function rename_remote()
   fi
 }
 
+function list_remotes()
+{
+  # check if the remote config file exists
+  if [ -f "$local_remote_config_file" ]; then
+    cat $local_remote_config_file
```

Comment:
Just `cat` a file does not look good. Let's copy `git remote -v` output, something like:
```
$kw remote --list
  origin root@something *
  ubu-tm root@anotherip
  arch-tm root@anotherxpto
```

In other words, you need to process this data a little bit.